content/html/concepts/semantic-html/terms/rt/rt.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: '<rt>'
+Description: 'Provides support for East Asian typography such as translations, pronunciation, or transliteration.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<rt>`** semantic HTML element represents the ruby text component of a ruby annotation, which provides translation, pronunciation, or transliteration information for East Asian typography. The `<rt>` element must be used within a `<ruby>` element at all times.
+
+## Example
+
+The following example provides a basic implementation of the tag illustrating a Romaji transliteration for the 'kenjutsu' characters within the `<ruby>` element:
+
+```pseudo
```

Comment:
@gs-Hani I think you meant to make this an html block rather than a pseudo block

Feel free to open an enhancement or edit PR